Updates for golden image heterogenous cluster support (#99811)
openshift/openshift-docs: modules/virt-creating-vm-instancetype.adoc

When configuring a downward-metrics device on an {ibm-z-name} system that uses a VM preference, set the `spec.preference.name` value to `rhel.9.s390x` or another available preference with the format `*.s390x`.
====

> . Filter the bootable volumes using the options provided. The *Architecture* filter is only available in heterogeneous clusters.

Reviewer: ```suggestion
. Heterogeneous clusters only: To filter the bootable volumes using the options provided, click *Architecture*.
```
[IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#tag)

Author: Fixed. Thank you!
